### PR TITLE
Vibrancy support

### PR DESCRIPTION
**Pull request by andrepolischuk**
@iamstarkov what do you think about it?

### CODE REVIEW

**Comment by iamstarkov** [comment]
do we need to encapsulate it into theme itself? i was thinking about standalone package

**Comment by andrepolischuk** [comment]
I think this variants are slightly different for standalone packages

**Comment by iamstarkov** [comment]
okay

**Comment by iamstarkov** [comment]
is hyper smart enough to not treat string with `/` as another package, which doesnt exist?

**Comment by andrepolischuk** [comment]
> is hyper smart enough to not treat string with `/` as another package, which doesnt exist?

I think yes. Hyper use commonjs `require` for plugins paths https://github.com/zeit/hyper/blob/master/app/plugins.js#L281-L301

**Comment by iamstarkov** [comment]
good

**Comment by iamstarkov** [comment]
do we need to add screenshot for that?

**Comment by iamstarkov** [comment]
look, just merged these in awesome-hyper
https://www.npmjs.com/package/hyper-sierra
https://www.npmjs.com/package/hyper-sierra-vibrancy

**Comment by andrepolischuk** [comment]
Do you think that we should be separate packages for one-light?

**Comment by iamstarkov** [comment]
i dont know

**Comment by iamstarkov** [comment]
wanted to let you know about other approaches

**Comment by iamstarkov** [comment]
but lets keep it simple

**Comment by iamstarkov** [comment]
feel free to merge

File: vibrancy.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+'use strict';
+const { decorateConfig } = require('./');
+const backgroundColor = 'rgba(250, 250, 250, 0.75)';
+
+exports.onWindow = browserWindow => browserWindow.setVibrancy('medium-light');
```
Comment (iamstarkov) [inline vibrancy.js]: can you describe be me what is a relation between semi-transparent color and vibrancy?

**Pull request merged**